<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListProyectosBase</class>
 <widget class="QWidget" name="ListProyectosBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listado de Proyectos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Genericos32x32/images/icons/proyecto_listado.png</normaloff>:/Genericos32x32/images/icons/proyecto_listado.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap>:/Genericos32x32/images/icons/proyecto_listado.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_crear">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Nueva amortizacion</string>
          </property>
          <property name="whatsThis">
           <string>Nueva amortizacion</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/document-new.png</normaloff>:/Images/document-new.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+N</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_borrar">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Borrar amortizacion</string>
          </property>
          <property name="whatsThis">
           <string>Borrar amortizacion</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:Images/edit-delete.png</normaloff>:Images/edit-delete.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_actualizar">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Borrar amortizacion</string>
          </property>
          <property name="whatsThis">
           <string>Borrar amortizacion</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/view-refresh.png</normaloff>:/Images/view-refresh.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="shortcut">
           <string>F5</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>261</width>
            <height>32</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="BcSubForm" name="mui_listado">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BcSubForm</class>
   <extends>QTableWidget</extends>
   <header>bcsubform.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages.qrc"/>
 </resources>
 <connections/>
</ui>
